.github/workflows/pr.yml

```diff
@@ -45,3 +46,39 @@ jobs:
             revert
             style
             test
+
+      - name: Get category label from PR title
+        id: get-category
+        env:
+          PR_TYPE: ${{ steps.pr-title.outputs.type }}
+        run: |
+          case "$PR_TYPE" in
+            "feat")
+              CATEGORY="C-enhancement"
+              ;;
+            "fix")
+              CATEGORY="C-bug"
+              ;;
+            "perf")
+              CATEGORY="C-enhancement"
```

Comment:
is this right? looks like we might check `perf` twice?

Reply:
i guess a pef change is both a chnage in performance and an enhancement?

Reply:
It's not correct. It should be C-performance. Good catch

Reply:
Fixed